main/exercise/question.class.php

```diff
@@ -2691,6 +2699,69 @@ public function countAnswers()
         return (int) $result['c'];
     }
 
+    /**
+     * Check if a question is shared between quices.
+     *
+     * @param int $questionId - question ID
+     *
+     * @return int - The number of quices where the question is used
+     */
+    public static function isQuestionOnOtherQuizs(int $questionId)
```

Comment:
The name of `isQuestionOnOtherQuizs()` is confusing because it really returns a number of quizzes it's in, not whether (true or false) it is in "other" quizzes. Should be named `countQuizzesUsingQuestion()`, for example.

Reply:
👌